src/scripts/tweaks/caught_up_line.js

```diff
@@ -0,0 +1,41 @@
+import { keyToCss } from '../../util/css_map.js';
+import { buildStyle } from '../../util/interface.js';
+import { onNewPosts } from '../../util/mutations.js';
+
+const hiddenClass = 'xkit-tweaks-caught-up-line-title';
+const borderClass = 'xkit-tweaks-caught-up-line-border';
+
+const styleElement = buildStyle(`
+  .${hiddenClass} > div { display: none; }
+  .${borderClass} > div > div { border-top: 4px solid rgb(var(--white-on-dark)); height: 0px; }
```

Comment:
i think it would be cleaner to just target the first `div` and set `overflow-y: hidden;` explicitly rather than relying on Tumblr's carousel CSS
```suggestion
  .${borderClass} > div {
    box-sizing: content-box;
    height: 0px;
    overflow-y: hidden;
    border-top: 4px solid rgb(var(--white-on-dark));
  }
```